entrypoint.sh

```diff
@@ -8,7 +8,8 @@ export MIXPANEL_TOKEN=$(gcloud secrets versions access latest --secret=MIXPANEL_
 export ETHERSCAN_API_KEY=$(gcloud secrets versions access latest --secret=ETHERSCAN_API_KEY)
 export ALCHEMY_API_KEY=$(gcloud secrets versions access latest --secret=ALCHEMY_API_KEY)
 export TELEGRAM_BOT_TOKEN=$(gcloud secrets versions access latest --secret=TELEGRAM_BOT_TOKEN)
+export DISCORD_BOT_TOKEN=$(gcloud secrets versions access latest --secret=DISCORD_BOT_TOKEN)
```

Comment:
If you can share the secret with me out of band, I will make sure this works

Reply:
will dm you in a few